spec/support/signon_helpers.rb

```diff
@@ -41,7 +41,7 @@ def self.available_user_count
   end
 
   def get_superuser
-    @@_superuser ||= User.new(superuser: true)
+    @_superuser ||= User.new(superuser: true)
```

Comment:
This strikes me as something that will change the behaviour - @cbaines does this look problematic? - I don't think any code in this repo actually runs this yet.

Reply:
Yeah, good spot Kevin. What I intended here was to access the 'class' variables, rather than the instance variables, which is what I think the `@@` does. I'm not quite sure how this interacts with the module, but the `next_user_number` variable in the User class should definitely be shared between instances, rather than being instance specific.

As for use, yes, this is used when you run the end-to-end tests with govuk-guix, as it runs the Rails apps using the Rails production environment, and using Signon, but this won't yet be run when using the docker-compose setup.

Reply:
@cbaines Thanks for the update, will just disable the rule for this class :+1: